lib/tasks/undercover.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'simplecov'
+require 'simplecov-lcov'
+require 'undercover'
+
+namespace :undercover do
+  desc "Runs undercover comparison against master"
+  task run_diff_master: :environment do
+    "bundle exec undercover --compare origin/master"
```

Comment:
What is the advantage of a rake task?

Reply:
I tried to follow the syntax or way we run tasks from `build.yml`, we seem to have opted for running them with bin/rake, for example:
```

        run: |
          bin/rake knapsack_pro:rspec

```

Reply:
Ah, yes. I think that's Knapsack specific. Before Knapsack, we used to run `bundle exec rspec`.